### PR TITLE
Use fonts for icons.

### PR DESCRIPTION
**Pull request by arcelena**

### CODE REVIEW

**Comment by arcelena** [comment]
Cause i did branch from 'master' where nothing exists on profile screen. As i know u r doing profile screen now. Do u want me to put icons anywhere on the profile screen?

**Comment by demkalkov** [comment]
Profile is already in master since yesterday

**Comment by arcelena** [comment]
Maybe better to merge this branch to master and then i'll create new branch for profile icons?

File: mobile/src/override/application-tabBar-icons.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { createIconSet } from 'react-native-vector-icons';
+
+const glyphMap = {
+    'calendar': 0xE002,
+    'feeds': 0xE005,
+    'helpdesk': 0xE007,
+    'people': 0xE00A,
+    'profile': 0xE00C
+};
+export const ApplicationTabbarIcons = createIconSet(glyphMap, 'aa-iconfont');
```
Comment (santee) [inline mobile/src/override/application-tabBar-icons.tsx]: rename to ApplicationIcon
1. It's not only tabbar.
2. It's a component name, so it has to be singular

**Comment by demkalkov** [comment]
**btw** I couldn't even run this branch on simulator because of some missing dependencies for  the `react-native-vector-icons` module. I've tried to reinstall packages and rebuild the app multiple times, but nothing helps

**Comment by arcelena** [comment]
It's aa-iconfont.ttf

**Comment by serjKim** [comment]
Why was Feather.ttf modified?

**Pull request merged**